src/main/java/org/myrobotlab/service/ProgramAB.java

```diff
@@ -1329,7 +1342,8 @@ public void onUtterance(Utterance utterance) throws Exception {
         // TODO: don't talk to bots.. it won't go well..
         // TODO: the discord api can provide use the list of mentioned users.
         // for now.. we'll just see if we see Mr. Turing as a substring.
-        if (utterance.text.contains(botName)) {
+        sleep = (sleep || utterance.text.contains("@")) && !utterance.text.contains(botName);

```

Comment:
i'd like to preserve existing behavior as default and make this behavior a configuration option exposed in the config

Reply:
agreed fixed